Sources/MongoSwift/WriteConcern.swift

```diff
@@ -127,7 +127,7 @@ public class WriteConcern: Codable {
             let journalStr = String(describing: journal)
             let wStr = String(describing: w)
             let timeoutStr = String(describing: wtimeoutMS)
-            throw MongoError.invalidArgument(message:
+            throw UserError.invalidArgumentError(message:
```

Comment:
let's update the docstring here to say this is the type of error thrown. 
same applies to the other places we've switched to this error but I won't duplicate the comment

Reply:
Done 👍